stylesheets/style.css

```diff
@@ -0,0 +1,91 @@
+* {
+    font: helvetica;
+    text-align: center;
+    /* border: 2px solid red; */
+
+}
+
+body {
+    margin: 0;
+    background-color: rgb(249, 198, 234);
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 10fr 1fr;
+    /* border: 5px solid blue; */
+}
+
+header {
+    background-color: rgb(8, 92, 95);
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+    /* border: 5px solid white; */
+}
+
+nav {
+    grid-column: 1 / span 1;
+    grid-row: 2 / span 1;
+    /* border: 5px solid rgb(69, 241, 129); */
+    display: flex;
+    justify-content: space-evenly;
+}
+
+main {
+    grid-column: 1 / span 1;
+    grid-row: 2 / span 2;
+    /* border: 5px solid rgb(66, 6, 6); */
+}
+
+footer {
+    background-color: rgb(8, 92, 95);
+    grid-column: 1 / span 1;
+    grid-row: 4 / span 1;
+    /* border: 5px solid rgb(139, 212, 243) */
+}
+
+footer > p {
+    color: white;
+}
+
+h1 {
+    color: white;
+    font-size: 150 rem;
+}
+
+h2 {
+    color: white;
+    font-size: xx-large;
+}
+
+h3 {
+    color: white;
+}
+
+h4 {
+    color: white;
+}
```

Comment:
Since these are the only properties, it would be DRY-er to do:
```
h3, h4 {
    color: white;
}
```
and adding in the other elements with just this one property.